.thoth.yaml

```diff
@@ -5,9 +5,9 @@ requirements_format: pipenv
 runtime_environments:
   - name: fc31
```

Comment:
can we change this as well, the name seems confusing, if we are using ubi

Reply:
thanks, https://github.com/thoth-station/qeb-hwt/pull/133/commits/ffbf5a4c180571f6aa19560aadfd6e65657e340e